pygeo/constraints/areaConstraint.py

```diff
@@ -290,6 +290,9 @@ def addConstraintsPyOpt(self, optProb):
                 wrt=self.getVarNames(),
             )
 
+    def writeTecplot(self, handle):
```

Comment:
Is this function overwritten by children classes?

Reply:
No it is not. This is here to make Python happy because this is an abstract method, meaning it must be implemented by child classes. In this case, I just make an empty implementation because there was no code previously, but in the future this should be implemented.

Reply:
Can we raise a not implemented error here?
It's confusing that the function exists but does nothing.

Reply:
This is a good point, I will update.